genBorder.js

```diff
@@ -0,0 +1,70 @@
+// TODO: Consider generating border classes as well that would just assign a default @neutral_silver
+// color. We need to be sure it would actually be useful before adding the extra CSS bloat.
+const _ = require("lodash");
+const fs = require("fs");
+
+
+const FILEPATH = "src/less/";
+const FILENAME = "border.less";
+const borderWidths = {
+  s: 0.0625, // 1px
+  m: 0.125, // 2px
+  l: 0.1875, // 3px
+};
```

Comment:
yeah, was thinking that too, but tested increasing the base font-size to see the effects and the borders growing along with the text actually didn't seem out of place at all. ends up working like an actual zoom.

that being said, not sure if that's actually what we want. seems fine, but will let @CarlNelson weigh in